### PR TITLE
Rock Paper Scissors Environment

### PR DESCRIPTION
**Pull request by harrisse**
I may refactor a little more but I believe this is now code complete.

### CODE REVIEW

File: kaggle_environments/envs/rps/rps.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        return state
+
+    step = len(env.steps)
+    player1.observation.step = step
+
+    def is_valid_action(player, max_sign):
+        return (
+            player.action is not None and
+            isinstance(player.action, int) and
+            0 <= player.action < max_sign
```
Comment (dster2) [inline kaggle_environments/envs/rps/rps.py:22]: nit: I'd interpret `max_sign` as inclusive, not exclusive

File: kaggle_environments/envs/rps/utils.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import math
+
+
+def get_score(left_move, right_move):
```
Comment (dster2) [inline kaggle_environments/envs/rps/utils.py:4]: Should this depend on `env.configuration.signs`?

File: kaggle_environments/envs/rps/rps.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        height = 400,
+        interactive,
+        isInteractive,
+        parent,
+        step,
+        update,
+        width = 400,
+    } = context;
+
+    const weapon_names = ["Rock", "Paper", "Scissors", "Spock", "Lizard"]
```
Comment (dster2) [inline kaggle_environments/envs/rps/rps.js:16]: nit: `sign` instead of `weapon` here too

File: kaggle_environments/envs/rps/utils.py (hunk 2 above)
Comment (harrisse) [reply kaggle_environments/envs/rps/utils.py:4]: Interestingly I don't think so, the modulo step handles "wrapping" around the edge of the signs length from what I understand, but this is just a direct port of Myles' work so he'd likely understand better.

File: kaggle_environments/envs/rps/rps.js (hunk 3 above)
Comment (harrisse) [reply kaggle_environments/envs/rps/rps.js]: Thanks, turns out I had "Find in *.py" checked so I missed all the non-py files 🤦

File: kaggle_environments/envs/rps/rps.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    const weapons_icons = ["👊", "📄", "✂️", "🖖", "🦎"]
```
[the remark this answers is not in the document]
Comment (harrisse) [reply kaggle_environments/envs/rps/rps.js]: Great idea, I'll pick two random ones somehow.

[the remark this answers is not in the document]
Comment (harrisse) [reply kaggle_environments/envs/rps/rps.js]: Or did you mean that you picked a tone per player randomly?

**Pull request merged**